tests/PackageDrupal8Test.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace DigipolisGent\Tests\Robo\Task\Package\Drupal8;
+
+use League\Container\ContainerAwareInterface;
+use League\Container\ContainerAwareTrait;
+use Robo\Common\CommandArguments;
+use Robo\Contract\ConfigAwareInterface;
+use Robo\Robo;
+use Robo\TaskAccessor;
+use Symfony\Component\Console\Output\NullOutput;
+
+class PackageDrupal8Test extends \PHPUnit_Framework_TestCase implements ContainerAwareInterface, ConfigAwareInterface
+{
+
+    use \DigipolisGent\Robo\Task\Package\Drupal8\loadTasks;
+    use TaskAccessor;
+    use ContainerAwareTrait;
+    use CommandArguments;
+    use \Robo\Task\Base\loadTasks;
+    use \Robo\Common\ConfigAwareTrait;
+
+    /**
+     * Set up the Robo container so that we can create tasks in our tests.
+     */
+    public function setUp()
+    {
+        $container = Robo::createDefaultContainer(null, new NullOutput());
+        $this->setContainer($container);
+        $this->setConfig(Robo::config());
+    }
+
+    /**
+     * Scaffold the collection builder.
+     *
+     * @return \Robo\Collection\CollectionBuilder
+     *   The collection builder.
+     */
+    public function collectionBuilder()
+    {
+        $emptyRobofile = new \Robo\Tasks();
+
+        return $this->getContainer()
+            ->get('collectionBuilder', [$emptyRobofile]);
+    }
+
+    public function testRun()
```

Comment:
Well there are files under the themes folder, and we check that there are no more files in the package than we expect with `$this->assertEmpty($archiveFiles);` at the end of the test.